packages/components/form/src/label-wrap.ts

```diff
@@ -27,15 +27,17 @@ export default defineComponent({
     updateAll: Boolean,
   },
   setup(props, { slots }) {
-    const el = ref<Nullable<HTMLElement>>(null)
-    const elForm = inject(elFormKey)
-    const elFormItem = inject(elFormItemKey)
+    const formContext = inject(formContextKey)
```

Comment:
Is `formContext` optional here?  LabelWrap or form might not work if not used in pair. So throwing an error faster might be a better choice?